regression/cbmc/r_w_ok_inconsistent_valid/test-global-no-cp.desc

```diff
@@ -0,0 +1,11 @@
+KNOWNBUG
```

Comment:
Why is this knownbug rather than future?

Reply:
This particular test made cbmc crash. I'll add a comment to say that, and will add a simpler test too that shows the same bug.